arangod/RocksDBEngine/RocksDBTransactionCollection.h

```diff
@@ -177,5 +178,8 @@ class RocksDBTransactionCollection : public TransactionCollection {
 
   bool _usageLocked;
   bool _exclusiveWrites;
+  std::variant<std::monostate, RocksDBMetaCollection::ExclusiveLock,
```

Comment:
nit: as we are now using std::variant in this header file, we should also add `#include <variant>` at the top of the file.